lib/main.js

```diff
@@ -3,17 +3,27 @@
 const EventEmitter = require('events');
 const express = require('express');
 const SinkMem = require('@asset-pipe/sink-mem');
-const JsReader = require('asset-pipe-js-reader');
-const CssReader = require('asset-pipe-css-reader');
-const body = require('body/json');
+const bundleJS = require('@asset-pipe/js-reader');
+const bundleCSS = require('@asset-pipe/css-reader');
 const boom = require('boom');
 const uuid = require('uuid/v4');
 const mime = require('mime-types');
 const { Transform } = require('readable-stream');
+const { promisify } = require('util');
+const body = promisify(require('body/json'));
```

Comment:
why? can't we just use `express.json()` middleware?

Reply:
It was there already...

Reply:
express bundles body-parser now, no reason at all to not use it IMO. you already change its usage in this PR anyways

could have joi as middleware as well, but maybe not in this PR

Reply:
Yea I think we should revisit and use joi in a few places to validate feeds

Reply:
was not drop-in-replacement as far as I can tell.

Reply:
https://github.com/asset-pipe/asset-pipe-build-server/issues/93